coalib/parsing/DefaultArgParser.py

```diff
@@ -187,23 +187,23 @@ def default_arg_parser(formatter_class=None):
                                metavar='ENUM',
                                help=MIN_SEVERITY_HELP)
 
-    # The following are "coala" specific arguments
-    if parser_type == 'coala':
-        SHOW_BEARS_HELP = ("Display bears and its metadata with the sections "
-                           "that they belong to")
-        outputs_group.add_argument('-B',
-                                   '--show-bears',
-                                   const=True,
-                                   action='store_const',
-                                   help=SHOW_BEARS_HELP)
+    SHOW_BEARS_HELP = ("Display bears and its metadata with the sections "
```

Comment:
If you remove the if condition, then this is even there for coala-format, coala-html. etc etc...
You want the if condition to check for coala and coala-json only